internal/process/vms.go

```diff
@@ -25,12 +25,10 @@ func createVirtualMachineSamples(config *config.Config) {
 			if vm.Summary.Runtime.Host == nil {
 				continue // This property is null if the virtual machine is not running and is not assigned to run on a particular host.
 			}
-			if _, ok := dc.Hosts[*vm.Summary.Runtime.Host]; !ok {
-				continue
-			}
-			if dc.Hosts[*vm.Summary.Runtime.Host].Parent == nil {
+			if h := dc.GetHost(vm.Summary.Runtime.Host.Reference()); h == nil || h.Parent == nil {
```

Comment:
if we have tagged the vms but not the host, i think this vm will be not reported.